app/helpers/check_answers_helper.rb

```diff
@@ -202,4 +202,14 @@ def questions
     questions ||= YAML.load_file(Rails.root.join("config/locales/en.yml"))
     questions["en"]["coronavirus_form"]["questions"].keys
   end
+
+  def base_path(field)
+    field.dig(:edit, :href).split("?").first
+  end
+
+  def question_answers(property, items)
+    items.select do |item|
+      item if property.dig(:items, :enum).include?("/#{base_path(item)}") && item[:value].present?
```

Comment:
This threw me a bit because I didn't realise for quite a while, that the property being passed in was a schema property, and that why we're checking against the enum.

Same goes for the `base_path` method.

Reply:
Thanks @leenagupte  👍  have changed approach.